src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/Router.java

```diff
@@ -99,16 +113,30 @@ public void init() {
         get("/platforms", platformResource::all);
         get("/platforms/:id", platformResource::get);
 
-        put("/populations", populationResource::put, Population.class);
-        get("/populations", populationResource::all);
-        get("/populations/:id", populationResource::get);
-        delete("/populations/:id", populationResource::delete);
+        put("/calibrations", calibrationResource::put, Calibration.class);
+        get("/calibrations", calibrationResource::all);
+        get("/calibrations/:id", calibrationResource::get);
+        delete("/calibrations/:id", calibrationResource::delete);
 
         get("/workers/:platform/identity", workerResource::identify);
         put("/workers", workerResource::put, Worker.class);
         get("/workers", workerResource::all);
         get("/workers/:id", workerResource::get);
         delete("/workers/:id", workerResource::delete);
+
+        get("/algorithms/task_chooser", algorithmResources::allTaskChoosers);
+        get("/algorithms/answer_quality", algorithmResources::allAnswerQualityAlgorithms);
+        get("/algorithms/rating_quality", algorithmResources::allRatingQualityAlgorithms);
```

Comment:
Shouldn't these be grouped in one REST call? And better use `-` than `_` in URLs.